### DISCUSSION TITLE
Empty state svg image for IoT - Microvisor (new product)

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#2397] [Request: Illustration] [answered]
Hello,

Can you please help us add Microvisor (brand new product) SVG for empty state illustration in to Fimga library as well as for using in the code?

Image that we require is -- https://library.twilio.com/l/15c16c5846ff7d55/ 
For past reference for similar request is - https://github.com/twilio-labs/paste/discussions/1544

Thank you

**Comment by TheSisb** [comment]
Hello @ishah-twilio,

I believe Bynder only contains PNGs. If you need a SVG, you will need to reach out to brand directly (#mkg-brand on Slack).

Out of curiosity, can you share where you need a different empty state illustration? What about the [current list of illustrations](https://paste.twilio.design/patterns/empty-state#examples-of-empty-state-use-cases) doesn't work?

**Reply by nkrantz** [reply]
Hey @ishah-twilio - checking in on this thread to see whether there are any updates or you're all set with the existing Empty State illustrations. Let me know!

**Comment by ishah-twilio** [comment]
Thank you for checking.

We still like this illustration (https://library.twilio.com/l/15c16c5846ff7d55/) to be added to Figma and the code to be used on the Overview page.

thank you.

**Reply by nkrantz** [reply]
Not sure if you saw Shadi's message above, but it would be helpful for us to have more context around why the current list of Empty State illustrations don't meet your needs?

**Comment by ishah-twilio** [comment]
I think as i said earlier, we would like to use Product appropriate illustration on the Overview page and carry it over to the empty stage of the product list.

In anycase, even if we don't use that illustration for the empty stage we will need it for the Overview page. Microvisor is a brand new product offering and none of its illustration are part of the code/Fimga.

**Comment by ishah-twilio** [comment, marked as answer]
As of now, this is not a high priority since Overview page will be not implemented in near future. So we can close this ticket and reopen in future.

**Reply by nkrantz** [reply]
Sounds good, I'll remove the "Active Discussion" label for now, and whenever you want to re-open this just comment on this thread!
